diff --git a/sites/all/modules/contrib/views_infinite_scroll/js/views_infinite_scroll.js b/sites/all/modules/contrib/views_infinite_scroll/js/views_infinite_scroll.js
index b1d1465..7f33c10 100644
--- a/sites/all/modules/contrib/views_infinite_scroll/js/views_infinite_scroll.js
+++ b/sites/all/modules/contrib/views_infinite_scroll/js/views_infinite_scroll.js
@@ -24,7 +24,7 @@ Drupal.behaviors.views_infinite_scroll = {
             var view_selector    = 'div.view-id-' + settings.view_name + '.view-display-id-' + settings.display;
             var content_selector = view_selector + ' > ' + settings.content_selector;
             var items_selector   = content_selector + ' ' + settings.items_selector;
-            var pager_selector   = view_selector + ' > div.item-list ' + settings.pager_selector;
+            var pager_selector   = view_selector + ' ' + settings.pager_selector;
             var next_selector    = view_selector + ' ' + settings.next_selector;
             var img_location     = view_selector + ' > div.view-content';
             var img_path         = settings.img_path;
